server/src/main/java/com/broadcom/lsp/cobol/core/preprocessor/delegates/GrammarPreprocessorListenerImpl.java

```diff
@@ -63,6 +64,7 @@
 public class GrammarPreprocessorListenerImpl extends CobolPreprocessorBaseListener
     implements GrammarPreprocessorListener {
 
+  private static final int AREA_A_POSITION = 7;
```

Comment:
Area A position is 8-11, so please rename this to "INDICATOR_POSITION"

Reply:
Nice! 👍